<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlSystemWindow</class>
 <widget class="QWidget" name="ControlSystemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlSystemDesinger</string>
  </property>
  <widget class="QTabWidget" name="BodePlot">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>0</y>
     <width>460</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Bode Plot</string>
    </attribute>
    <widget class="QCustomPlot" name="magnitudePlot" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
    <widget class="QCustomPlot" name="phasePlot" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>451</width>
       <height>200</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="tfLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>650</y>
     <width>136</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Transfer Function</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>690</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="plotButton">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save Controller</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="RootLocus">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>0</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Root Locus</string>
    </attribute>
    <widget class="QCustomPlot" name="rootlocusPlot" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="StepResponse">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>300</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Step Response</string>
    </attribute>
    <widget class="QCustomPlot" name="stepresponsePlot" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>441</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
